src/PrestaShopBundle/Command/SecurityAnnotationLinterCommand.php

```diff
@@ -55,6 +55,29 @@ final class SecurityAnnotationLinterCommand extends Command
      */
     private $securityAnnotationLinter;
 
+    /**
+     * @var array
+     */
+    private const EXCEPTION_ROUTES = [
+        'admin_common_notifications',
+        'admin_common_notifications_ack',
+        'admin_common_pagination',
+        'admin_common_sidebar',
+        'admin_common_reset_search',
+        'admin_common_reset_search_by_filter_id',
+        'admin_security_compromised',
+        'admin_get_ajax_categories',
+        'admin_product_list', // Deprecated products v1
```

Comment:
These ones are deprecated but still effective in 8.1 since the v2 can be disabled So they should be fixed as well Even though they've been incorrect for a long time since we aim at fixing all required routes they should also be fixed

Reply:
> since we aim at fixing all required routes they should also be fixed

We aim to fix all required routes, but not in this PR 😉 . This PR must unblock the 8.1.0 release process, not fix every problem in PrestaShop.

I know you'll disagree ^^ but I think we should aim, in this Pull Request, to fix _the new routes introduced in 8.1.0_ and nothing else. Another PR can be done later to fix years-long problems, but I don't want this PR to have a scope too large. Because I know from experience that PRs with too large scopes take weeks to be reviewed, tested and merged.